NSSCodeForNashville/NashvilleHistory
# Ddw all places

## Status
**READY/IN DEVELOPMENT**

## Documentation

[ X] I have fully documented features in this PR's code. Please put an `X` in the box to confirm.

[ ] This PR does **NOT** require documentation.

## Description

The goal of this pull request was to create a view with all of the historical and art markers from the database. 

## Resolves Issue Number

#30 

## Deploy Notes
Notes regarding deployment the contained body of work. 

```sh
git pull --prune
git checkout ddw-all-places
ionic serve --lab
```

## Steps to Test or Reproduce

1. Go to the branch and then to the All Places item in the side menu
1. Make sure that the filters (art, historical, and civil war site) are working

## Impacted Files in Application

* AllPlacesFact.js
* AllPlacesCtrl.js
* all-places.html
